omero/developers/Web/CreateApp.txt

```diff
@@ -9,6 +9,8 @@ database mapping, since all data comes from the OMERO server and is
 saved back there. You will notice that the models.py files in each app
 are empty.
 
+All OMERO applications can now be installed from :pypi:`Python Package Index <>`.
```

Comment:
Perhaps drop the "now" since this page isn't date, and do we want to say something like "core" or "official"? i.e. all the ones that really want someone to use will by on PyPI though there may be others out there.